analyzer/restapi-plugin/src/main/java/eu/fasten/analyzer/restapiplugin/Main.java

```diff
@@ -0,0 +1,43 @@
+package eu.fasten.analyzer.restapiplugin;
+
+import eu.fasten.server.connectors.PostgresConnector;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
+
+import java.sql.SQLException;
+
+@CommandLine.Command(name = "RestAPIPlugin")
+public class Main implements Runnable{
+
+    private static Logger logger = LoggerFactory.getLogger(Main.class);
+
+    @CommandLine.Option(names = {"-d", "--database"},
+            paramLabel = "dbURL",
+            description = "Database URL for connection",
+            defaultValue = "jdbc:postgresql:postgres")
+    String dbUrl;
+
+    @CommandLine.Option(names = {"-u", "--user"},
+            paramLabel = "dbUser",
+            description = "Database user name",
+            defaultValue = "postgres")
+    String dbUser;
+
+    public static void main(String[] args) {
+        final int exitCode = new CommandLine(new Main()).execute(args);
```

Comment:
Actually, I just updated to:
`new CommandLine(new Main()).execute(args);`
Just like the POManalyzer, since the intention of this plugin is to persist.